### PR TITLE
Magnum: Manage Clusters Get/List

### PR DESCRIPTION
**Pull request by JackKuei**
Prior to a PR being reviewed, there needs to be a Github issue that the PR
addresses. Replace the brackets and text below with that issue number.

For #[1062 [Migrate Magnum Branch Onto Master]](https://github.com/gophercloud/gophercloud/issues/1062)

Links to the line numbers/files in the OpenStack source code that support the
code in this PR:

[[clusters:get]](https://github.com/openstack/magnum/blob/master/magnum/api/controllers/v1/cluster.py#L382)
[[clusters:list]](https://github.com/openstack/magnum/blob/master/magnum/api/controllers/v1/cluster.py#L318)

### CODE REVIEW

**Comment by JackKuei** [comment]
@ozerovandrei Was going to remove the [WIP] after passed all the checks, but didn't get a chance. Removed the [WIP] now, let me know if you see something.

**Comment by JackKuei** [comment]
@jtopjian This one is ready to be reviewed

**Comment by ozerovandrei** [comment]
recheck

File: openstack/containerinfra/v1/clusters/urls.go [hunk 1]
```diff
@@ -13,3 +13,14 @@ func commonURL(client *gophercloud.ServiceClient) string {
 func createURL(client *gophercloud.ServiceClient) string {
 	return commonURL(client)
 }
+func idURL(client *gophercloud.ServiceClient, id string) string {
```
Comment (jtopjian) [inline openstack/containerinfra/v1/clusters/urls.go]: I think this can be removed?

**Comment by JackKuei** [comment]
@jtopjian Ready to be merged

**Comment by jtopjian** [comment]
LGTM - thank you!

**Pull request merged**